src/schemaPostgres.ts

```diff
@@ -174,9 +174,19 @@ export class PostgresDatabase {
     const fkeys = foreignKeys[tableName] || {};
 
     await this.db.each<T>(
-      'SELECT column_name, udt_name, is_nullable, column_default IS NOT NULL as has_default ' +
-        'FROM information_schema.columns ' +
-        'WHERE table_name = $1 and table_schema = $2',
+      `
+        SELECT
+          attname as column_name,
+          pt.typname as udt_name,
+          CASE WHEN a.attnotnull THEN 'NO' ELSE 'YES' END as is_nullable,
+          pg_get_expr(ad.adbin, ad.adrelid) is not null as has_default
```

Comment:
I looked at the definition of `information_schema.columns` to get this "pg_get_expr" call, same semantics there with a left join -- I don't understand it, but seems to pass integration tests :)